src/main/scala/cromwell/engine/db/slick/ExecutionComponent.scala

```diff
@@ -84,6 +85,27 @@ trait ExecutionComponent {
       if workflowExecution.workflowExecutionUuid === workflowExecutionUuid
     } yield execution)
 
+  val executionsByWorkflowExecutionIdAndCallKey = Compiled(
```

Comment:
Will this ever return more than one execution? Also I slightly disagree with the "CallKey" in the name, as no argument is a CallKey.